content/docs/guides/prisma.md

```diff
@@ -96,12 +96,33 @@ When you finish updating your `.env` file, your variable settings should appear
 
 ```ini shouldWrap
 # Pooled Neon connection string
-DATABASE_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456-pooler.us-east-2.aws.neon.tech/dbname?sslmode=require&pgbouncer=true"
+DATABASE_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456-pooler.us-east-2.aws.neon.tech/dbname?sslmode=require"
 
 # Unpooled Neon connection string
 DIRECT_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname?sslmode=require"
 ```
 
+#### Using a pooled connection with Prisma Migrate
+
+With Prisma ORM 5.10 or higher, you can use a pooled Neon connection string with Prisma Migrate. In this case, you only need to define the pooled connection string in your `schema.prisma` file. Adding a `directUrl` property to the datasource block in your `schema.prisma` file and defining a `DIRECT_URL` setting in your environment file are not required. Your complete configuration will look like this:
```

Comment:
Does the directURL part need to be mentioned at all? If it was a prior workaround that no longer applies, it's probably not needed now. But if it's a common configuration for other integrations though, it's good.

Reply:
yeah, it would be common for any using Prisma Migrate with Neon. I think we can leave it.